rct229/ruleset_functions/normalize_interior_lighting_schedules.py

```diff
@@ -0,0 +1,97 @@
+from rct229.data_fns.table_G3_7_fns import table_G3_7_lookup
+from rct229.utils.assertions import assert_, assert_required_fields, getattr_
+from rct229.utils.jsonpath_utils import find_exactly_one_with_field_value
+
+# Intended for internal use
+from rct229.utils.pint_utils import ZERO
+
+GET_NORMALIZE_SPACE_SCHEDULE__REQUIRED_FIELDS = {
+    "space": {
+        "$": ["lighting_space_type", "floor_area"],
+        "interior_lighting[*]": ["lighting_multiplier_schedule"],
+    }
+}
+
+
+def normalize_interior_lighting_schedules(space, space_height, schedules):
+    """This function would determine a normalized schedule for a data element in space.
+    NOTE: The function currently only works for interior lighting
+    Parameters
+    ----------
+    space: JSON - The space element that needs to determine a normalized schedule, e.g. space.interior_lighting
+    space_height: float, height of a space, it is typically the average height of the thermal zone
+    schedules: List[JSON] - schedule element
+    Returns
+    -------
+    A list containing 8760/8784 hourly values of a noralized schedule of the space data element
+    """
+    assert_required_fields(
+        GET_NORMALIZE_SPACE_SCHEDULE__REQUIRED_FIELDS["space"], space
+    )
+
+    space_total_power_per_area = 0.0
```

Comment:
This should be `space_total_power_per_area = ZERO.POWER_PER_AREA`.  I'm surprised pint doesn't complain on line 42.